ui/app/components/auth-form.js

```diff
@@ -166,7 +166,7 @@ export default Component.extend(DEFAULTS, {
     return templateName;
   }),
 
-  hasCSPError: alias('csp.connectionViolations.firstObject'),
+  hasCSPError: alias('csp.connectionViolations'),
```

Comment:
Look at the computer property connectionViolations to see how this now works. This is the only use case of connectionViolations.

Reply:
Looking at the computed property it uses `filter` which returns an empty array if the condition isn't met. This means that `hasCSPError` will always be truthy. 

So I think this parameter should be updated to be a computed property that returns a boolean whether or not `csp.connectionViolations` has a length or not.
```js
  connectionViolations: computed('events.@each.violatedDirective', function () {
    return this.events.filter((e) => e.violatedDirective.startsWith('connect-src'));
  }),
```

Reply:
Nvm! I see you updated this getter below. Nice!